test/pobox_give_away_SUITE.erl

```diff
@@ -0,0 +1,88 @@
+-module(pobox_give_away_SUITE).
+-include_lib("common_test/include/ct.hrl").
+-compile(export_all).
+
+all() -> [
+  owner_gives_away_ownership,
+  owner_gives_away_ownership_with_data,
+  owner_gives_away_ownership_to_dead_process,
+  foreign_process_tries_to_gives_away_ownership,
+  give_away_does_not_change_heir
+].
+
+owner_gives_away_ownership(_Config) ->
+    {ok, Box} = pobox:start_link([]),
+    Self = self(),
+    Func = fun() ->
+      Fn = fun(F) -> receive X -> Self ! X end, F(F) end,
+      Fn(Fn)
+    end,
+    H = erlang:spawn(Func),
+    true = pobox:give_away(Box, H, 5000),
+    true = receive
+        {pobox_transfer, Box, Self, undefined, give_away} -> true;
+        Other -> Other
+    after
+      5000 -> false
+    end.
```

Comment:
If you call:
```erlang
-include_lib("stdlib/include/assert.hrl"). % after OTP18 I think?
-include_lib("eunit/include/eunit.hrl"). % always valid
```
Then you'll be able to replace `true = <exp>` with `?assert(<Exp>)` and get fancier reporting when it fails.

Reply:
Thanks for the tip. I will make the change.

Reply:
Done.